lib/modules/manager/bundler/extract.spec.ts

```diff
@@ -175,6 +175,7 @@ describe('modules/manager/bundler/extract', () => {
       gem "inline_source_gem", source: 'https://gems.foo.com'
       gem 'inline_source_gem_with_version', "~> 1", source: 'https://gems.bar.com'
       gem 'inline_source_gem_with_variable_source', source: baz
+      gem 'inline_source_gem_with_variable_source_and_require_after', source: baz, require: %w[inline_source_gem]
```

Comment:
Without the change, this would parse the source as `baz, require: %w[inline_source_gem]` :/